### PR TITLE
add orchestrator network endpoints

### PR DESCRIPTION
**Pull request by hkaj**
### What does this PR do?

add orchestrator network endpoints to the docs

### Preview

https://docs-staging.datadoghq.com/haissam/add-orchestrator-network-endpoint

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/network.md [hunk 1]
```diff
@@ -56,7 +58,8 @@ All of these domains are **CNAME** records pointing to a set of static IP addres
 
   - [APM][1] data is `trace.agent.datadoghq.eu`
   - [Live Containers][2] data is `process.datadoghq.eu`
-  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][5] for more information.
+  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.com` in HTTP. Review the list of [logs endpoints][6] for more information.
```
Comment (sarina-dd) [inline content/en/agent/guide/network.md:61]:
```suggestion
  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][6] for more information.
```

File: content/en/agent/guide/network.md [hunk 2]
```diff
@@ -56,7 +58,8 @@ All of these domains are **CNAME** records pointing to a set of static IP addres
 
   - [APM][1] data is `trace.agent.datadoghq.eu`
   - [Live Containers][2] data is `process.datadoghq.eu`
-  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][5] for more information.
+  - [Logs][3] data is `agent-intake.logs.datadoghq.eu` for TCP traffic, `agent-http-intake.logs.datadoghq.eu` in HTTP. Review the list of [logs endpoints][6] for more information.
+  - [Orchestrator Resources][4] data is `orchestrator.datadoghq.com`
```
Comment (sarina-dd) [inline content/en/agent/guide/network.md:62]: @hkaj Wouldn't this be .eu? Want to verify before committing a change.

Also just a heads up, there was a weird merge commit error with master as we just pushed changes this afternoon to this same doc. Added what was in that PR to this PR so we don't overwrite those changes.

**Comment by sarina-dd** [comment]
@hkaj Just a heads up: we had to make additional changes today to this specific doc page as we were running into partial shortcode errors. A lot of this doc has now changed to include the new partial shortcode. I'm going to pull in the new changes before merging.

File: content/en/agent/guide/network.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ further_reading:
   - [APM][1] data is `trace.agent.`{{< region-param key="dd_site" code="true" >}}
   - [Live Containers][2] data is `process.`{{< region-param key="dd_site" code="true" >}}
   - [Logs][3] data is `agent-intake.logs`{{< region-param key="dd_site" code="true" >}} for TCP traffic, `agent-http-intake.logs.`{{< region-param key="dd_site" code="true" >}} in HTTP. Review the list of [logs endpoints][4] for more information.
+  - [Orchestrator Resources][5] data is `orchestrator.datadoghq.com`
```
Comment (sarina-dd) [inline content/en/agent/guide/network.md:29]: Just need confirmation if this is `.com` for both US & EU. If it changes, I can add the partial which will change this to `.eu` whenever someone uses the EU region selector.

Comment (hkaj) [reply content/en/agent/guide/network.md:29]: good catch, it's .com for US et .eu for UE.

File: content/en/agent/guide/network.md [hunk 4]
```diff
@@ -25,7 +25,8 @@ further_reading:
 
   - [APM][1] data is `trace.agent.`{{< region-param key="dd_site" code="true" >}}
   - [Live Containers][2] data is `process.`{{< region-param key="dd_site" code="true" >}}
-  - [Logs][3] data is `agent-intake.logs.`{{< region-param key="dd_site" code="true" >}} for TCP traffic, `agent-http-intake.logs.`{{< region-param key="dd_site" code="true" >}} in HTTP. Review the list of [logs endpoints][4] for more information.
+  - [Logs][3] data is `agent-intake.logs`{{< region-param key="dd_site" code="true" >}} for TCP traffic, `agent-http-intake.logs.`{{< region-param key="dd_site" code="true" >}} in HTTP. Review the list of [logs endpoints][4] for more information.
+  - [Orchestrator Resources][5] data is `orchestrator.datadoghq.com`
```
Comment (sarina-dd) [inline content/en/agent/guide/network.md:29]:
```suggestion
  - [Orchestrator Resources][5] data is `orchestrator.`{{< region-param key="dd_site" code="true" >}}.
```

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**